## Exemplo 1

Uma estrutura será contruída em madeira serrada, classe C30, cuja unidade de equilíbrio é de 14%, em local com umidade relativa do ar de 70%. 
O carregamento será de longa duração.
Determine as tensões normais resistentes de cálculo paralela e normal às fibras e de cisalhamento.

- Solução:

Tensões normais resistentes de cálculo paralela e normal às fibras:
 
 - $f_{t,0,d}$: Tensão de tração resistente de cáculo paralela às fibras;
 - $f_{t,90,d}$: Tensão de tração resistente de cáculo perpendicular às fibras;
 - $f_{c,0,d}$: Tensão de compressão resistente de cáculo paralela às fibras;
 - $f_{c,90,d}$: Tensão de compressão resistente de cáculo perpendicular às fibras;

Tensão de cisalhamento resistentes de cálculo: $f_{v,d}$

$X_{d} = k_{mod} \frac{X_{k}}{\gamma_{w}}$

$k_{mod} = k_{mod_{1}} \times k_{mod_{2}}$

Calculando $k_{mod}$:

Pela Tabela 4 da ABNT NBR 7190(1):2023, para carregamento de longa duração e madeira serrada, $k_{mod_{1}} = 0.70$.
Como unidade de equilíbrio é de 14% em local com umidade relativa do ar de 70%, a umidade é de classe II. 
Ainda pela mesma norma, Tabela 5, para classe de umidade II, $k_{mod_{2}} = 0.90$, para madeira serrada. Logo:


In [1]:
k_mod1 = 0.70
k_mod2 = 0.90
k_mod = k_mod1 * k_mod2
k_mod

0.63

In [2]:
gama_w = 1.4

Para a madeira de classe C30, da Tabela 3, ABNT NBR 7190(1):2023, obtemos:

In [3]:
f_t0k = 18.0
f_t90k = 0.4
f_c0k = 23.0
f_c90k = 2.7
f_vk = 4.0

In [4]:
f_t0d = k_mod * f_t0k / gama_w
f_t90d = k_mod * f_t90k / gama_w
f_c0d = k_mod * f_c0k / gama_w
f_c90d = k_mod * f_c90k / gama_w

# Nas linhas abaixo, imprimo os resultados, não se preocupe com sintaxe.

print(f"f_t0d = {f_t0d:.2f} MPa")
print(f"f_t90d = {f_t90d:.2f} MPa")
print(f"f_c0d = {f_c0d:.2f} MPa")
print(f"f_c90d = {f_c90d:.2f} MPa")

f_t0d = 8.10 MPa
f_t90d = 0.18 MPa
f_c0d = 10.35 MPa
f_c90d = 1.22 MPa


Para tensões de cisahamento, alteramos, $\gamma_{w} = 1.8$.

In [5]:
gama_w = 1.8
f_vd = k_mod * f_vk / gama_w
print(f"f_vd = {f_vd:.2f} MPa")

f_vd = 1.40 MPa


Então, os parâmetros solicitados são:

$$
f_{t,0,d} = 8.10 MPa 
$$

$$
f_{t,90,d} = 0.18 MPa 
$$

$$
f_{c,0,d} = 10.35 MPa 
$$

$$
f_{c,90,d} = 1.22 MPa 
$$

$$
f_{v,d} = 1.40 MPa
$$